reporter/src/funTest/assets/evaluated-model-reporter-test-expected-output.json

```diff
@@ -0,0 +1,708 @@
+    "scope" : 1,
+    "path" : [ ]
+  }, {
+    "_id" : 5,
+    "pkg" : 5,
+    "project" : 0,
+    "scope" : 1,
+    "path" : [ 2 ]
+  } ],
+  "dependency_trees" : [ {
+    "key" : 0,
+    "title" : "Gradle:com.here.ort.gradle.example:lib:1.0.0",
+    "pkg" : 0,
+    "children" : [ {
+      "key" : 1,
+      "title" : "compile",
+      "children" : [ {
+        "key" : 2,
+        "title" : "Maven:org.apache.commons:commons-text:1.1",
+        "pkg" : 4,
+        "children" : [ {
+          "key" : 3,
+          "title" : "Maven:org.apache.commons:commons-lang3:3.5",
+          "pkg" : 3,
+          "children" : [ ]
+        } ]
+      } ]
+    }, {
+      "key" : 4,
+      "title" : "testCompile",
+      "children" : [ {
+        "key" : 5,
+        "title" : "Ant:junit:junit:4.12",
+        "pkg" : 2,
+        "children" : [ {
+          "key" : 6,
+          "title" : "Maven:org.hamcrest:hamcrest-core:1.3",
+          "pkg" : 5,
+          "children" : [ ]
+        } ]
+      }, {
+        "key" : 7,
+        "title" : "Maven:org.apache.commons:commons-text:1.1",
+        "pkg" : 4,
+        "children" : [ {
+          "key" : 8,
+          "title" : "Maven:org.apache.commons:commons-lang3:3.5",
+          "pkg" : 3,
+          "children" : [ ]
+        } ]
+      } ]
+    } ]
+  }, {
+    "key" : 9,
+    "title" : "Gradle:com.here:nested-fake-project:1.0.0",
+    "pkg" : 1,
+    "path_excludes" : [ 0 ],
+    "children" : [ ]
+  } ],
+  "violation_resolutions" : [ {
+    "_id" : 0,
+    "message" : "Apache-2.0 hint",
+    "reason" : "CANT_FIX_EXCEPTION",
+    "comment" : "Apache-2 is not an issue."
+  } ],
+  "violations" : [ {
+    "_id" : 0,
+    "rule" : "rule 1",
+    "pkg" : 2,
+    "license" : 5,
+    "license_source" : "DETECTED",
+    "severity" : "ERROR",
+    "message" : "EPL-1.0 error",
+    "how_to_fix" : "* *Step 1*\n* __Step 2__\n* ***Step 3***\n```\nSome long text verify that overflow:scroll is working as expected.\n```",
+    "resolutions" : [ ]
+  }, {
+    "_id" : 1,
+    "rule" : "rule 2",
+    "pkg" : 4,
+    "license" : 2,
+    "license_source" : "DECLARED",
+    "severity" : "HINT",
+    "message" : "Apache-2.0 hint",
+    "how_to_fix" : "* *Step 1*\n* __Step 2__\n* ***Step 3***\n```\nSome long text verify that overflow:scroll is working as expected.\n```",
+    "resolutions" : [ 0 ]
+  }, {
+    "_id" : 2,
+    "rule" : "rule 3",
+    "pkg" : 5,
+    "license" : 8,
+    "license_source" : "CONCLUDED",
+    "severity" : "WARNING",
+    "message" : "BSD-3-Clause warning",
+    "how_to_fix" : "* *Step 1*\n* __Step 2__\n* ***Step 3***\n```\nSome long text verify that overflow:scroll is working as expected.\n```",
+    "resolutions" : [ ]
+  } ],
+  "declared_license_stats" : {
+    "Apache-2.0" : 2,
+    "BSD-3-Clause" : 1,
+    "EPL-1.0" : 1
+  },
+  "detected_license_stats" : {
+    "Apache-2.0" : 1,
+    "Apache-2.0 (multi-line)" : 1,
+    "Apache-2.0 (single-line)" : 1,
+    "MIT" : 1
+  },
+  "statistics" : {
```

Comment:
Consider moving the `stats` to the top as it is kind of a `summary`.

Reply:
I will do that in the separate PR which will also integrate the license stats with the `Statistics`.